src/cow-react/modules/limitOrders/utils/getOrderSurplus.ts

```diff
@@ -11,62 +10,146 @@ type Surplus = {
 }
 
 /**
- * Calculates SELL surplus based on buy amounts
+ * Calculates SELL surplus
  *
- * @param buyAmount buyAmount
- * @param executedBuyAmount executedBuyAmount
  * @returns Sell surplus
  */
-export function getSellSurplus(buyAmount: BigNumberish, executedBuyAmount: BigNumberish): Surplus {
+export function getSellSurplus(order: Order): Surplus {
```

Comment:
That's true and a fair point.
I'll do that on Explorer then, since the base behaviour is already tested there https://github.com/cowprotocol/explorer/blob/80b387605bde2020c0be938f9417a984063bdb40/test/utils/operator/orderSurplus.test.ts#L15

Reply:
Exact same logic copied over to Explorer with unittests https://github.com/cowprotocol/explorer/pull/414